test/common/ssl/context_impl_test.cc

```diff
@@ -18,6 +21,24 @@ namespace Ssl {
 
 class SslContextImplTest : public SslCertsTest {};
 
+class TestSecretImpl : public Secret::Secret {
+public:
+  TestSecretImpl(const envoy::api::v2::auth::Secret& message, const std::string name)
+      : message_([&message] {
+          envoy::api::v2::auth::Secret cfg;
+          cfg.CopyFrom(message);
+          return cfg;
+        }()),
```

Comment:
can it be simplified as message_(message)

Reply:
Replaced